Encounter Me.Api/Controllers/TokenController.cs

```diff
@@ -73,8 +73,8 @@ private async Task<dynamic> GenerateToken(string username)
 
             var claims = new List<Claim>
             {
-                new Claim(ClaimTypes.Name, username),
-                new Claim(ClaimTypes.NameIdentifier, user.FirstName), //should be user id. 
+                new Claim(ClaimTypes.Name, user.UserName),
+                new Claim(ClaimTypes.UserData, user.Id.ToString()), //should be user id. 
```

Comment:
Since it is code clean up you could have deleted the comment too.

Reply:
Cleanup only involved xp and trail following scripts

Reply:
Sorry, got a bit confused due to PR name then.